src/config.h

```diff
@@ -79,7 +80,8 @@ struct UptaneConfig {
         primary_ecu_hardware_id(""),
         director_server(""),
         repo_server(""),
-        key_source(kFile) {}
+        key_source(kFile),
+        key_type(kRSA2048) {}
```

Comment:
I think we need to be able to set this via the config file. It needs to be handled in `Config::updateFromPropertyTree` and `Config::writeToFile`.